src/components/home/XinDescription.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const XinDescription = () => {
+  return (
+    <div className="box-sizing w-full border-y-[2vh] border-csa-tan-300 bg-csa-green-100">
```

Comment:
for super small things like that border, you can use direct values like border-y-2 instead of vh values.
```suggestion
    <div className="border-y-8 border-csa-tan-300 bg-csa-green-100">
```